config/application.rb

```diff
@@ -44,5 +44,10 @@ class Application < Rails::Application
     config.active_record.encryption.primary_key = ENV.fetch("AR_ENCRYPTION_PRIMARY_KEY", "fake-primary-key")
     config.active_record.encryption.deterministic_key = ENV.fetch("AR_ENCRYPTION_DETERMINISTIC_KEY", "fake-deterministic-key")
     config.active_record.encryption.key_derivation_salt = ENV.fetch("AR_ENCRYPTION_KEY_DERIVATION_SALT", "fake-key-derivation-salt")
+
+    # HMRC interface api connection details
+    config.x.hmrc_interface.host = ENV.fetch("HMRC_API_HOST", nil)
```

Comment:
Not a blocker, as it will mean a lot of renaming but while I see the same naming convention is used on Apply, since the actual "HRMC API" is separate from the HMRC Interface for me it would be clearer if they were called.

```
config.x.hmrc_interface.host = ENV.fetch("HMRC_INTERFACE_HOST", nil)
...etc
```